include/aklite.h

```diff
@@ -0,0 +1,178 @@
+// Copyright (c) 2021 Foundries.io
+// SPDX-License-Identifier: Apache-2.0
+
+#include <string>
+
+#include <boost/filesystem.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include "json/json.h"
+
+class LiteClient;
+
+/**
+ * A high-level representation of a TUF Target in terms applicable to a
+ * FoundriesFactory.
+ */
+class TufTarget {
+ public:
+  TufTarget(std::string name, std::string sha256, int version, Json::Value custom)
+      : name_(std::move(name)), sha256_(std::move(sha256)), version_(version), custom_(std::move(custom)) {}
+
+  /**
+   * Return the TUF Target name. This is the key in the targets.json key/value
+   * singed.metadata dictionary.
+   */
+  std::string Name() const { return name_; }
```

Comment:
I suppose the getters could return `const std::string&`?

Reply:
ccb842e4586a4c67cffbac69a80a0aff8c1f0238